README.md

```diff
@@ -1,24 +1,57 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# TomBudge
 
-Things you may want to cover:
+- Mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what
 
-* Ruby version
 
-* System dependencies
+## Built With
 
-* Configuration
+- Rails
 
-* Database creation
+## Getting Started
 
-* Database initialization
+- Install rails app 
+    rails new project
+-Run the app
+     rails s
+- Open your default browser and run
+    http://127.0.0.1:3000
 
-* How to run the test suite
+- Create a repository
+- clone it using "git@github.com:gasangw/Budget-app.git"
 
-* Services (job queues, cache servers, search engines, etc.)
+## Live Demo
 
-* Deployment instructions
+- [link](https://immense-coast-55009.herokuapp.com/)
```

Comment:
- [ ] Please update your live demo link to the latest version of your app because it crashes when I try to sign up

Reply:
Solved